### PR TITLE
DGC 273 Contrast Issues

### PR DESCRIPTION
**Pull request by charlieartist**
Color contrast issues addressed in the main navigation, by increasing opacity, both for default and active states on navigation elements, as well as the text in hover state. All contrasts pass AA for large text.

In addition, a subtle drop shadow was added to the active and hover states to boost contrast with the surrounding elements.

### CODE REVIEW

File: docroot/themes/custom/twentyeighteen/sass/components/navigation/_main-menu.scss [hunk 1]
```diff
@@ -156,7 +156,7 @@ input[type="checkbox"] + label.toggle::before {
 
   a {
     display: block;
-    color: $med-gray;
+    color: rgba(255, 255, 255, 0.75);
```
Comment (lkopacz) [inline docroot/themes/custom/twentyeighteen/sass/components/navigation/_main-menu.scss]: Please use `color: rgba($white, 0.75);`

File: docroot/themes/custom/twentyeighteen/sass/components/navigation/_main-menu.scss [hunk 2]
```diff
@@ -167,7 +167,14 @@ input[type="checkbox"] + label.toggle::before {
   }
 
   a:hover {
-    color: $med-gray;
+    color: rgba(254, 236, 137, 0.875);
+    text-shadow: 0 0 0.25em rgba(0, 0, 0, 0.75);
```
Comment (lkopacz) [inline docroot/themes/custom/twentyeighteen/sass/components/navigation/_main-menu.scss]: Please use `rgba($blk, 0.75)`

File: docroot/themes/custom/twentyeighteen/sass/components/navigation/_main-menu.scss [hunk 3]
```diff
@@ -167,7 +167,14 @@ input[type="checkbox"] + label.toggle::before {
...
+    text-decoration: none;
+  }
+
+  a.is-active {
+    color: rgba(255, 255, 255, 1);
```
Comment (lkopacz) [inline docroot/themes/custom/twentyeighteen/sass/components/navigation/_main-menu.scss]: Please use `color: $white;`

File: docroot/themes/custom/twentyeighteen/sass/components/navigation/_main-menu.scss [hunk 4]
```diff
@@ -167,7 +167,14 @@ input[type="checkbox"] + label.toggle::before {
...
+    text-shadow: 0 0 0.25em rgba(0, 0, 0, 0.75);
```
Comment (lkopacz) [inline docroot/themes/custom/twentyeighteen/sass/components/navigation/_main-menu.scss]: Please use `rgba($blk, 0.75)`

**Comment by charlieartist** [comment]
@mikemadison13 Due to this branch being out-of-date, it was recreated in this [pull request](https://github.com/Drupal4Gov/Drupal-GovCon-2017/pull/674), which was successfully merged.

**Pull request closed without merging**